### PR TITLE
Feature/StyleUpdates

### PR DESCRIPTION
**Pull request by SandraCoburn**
# Description
Made changes to Parent components to update styles. Less files were updates. Added mixins for reusable styles. Added a variable to theme overrides to change background to white to parent components.

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [X] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [X] No
- [ ] Not necessary

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

**Review by bramirez96 — APPROVED** [review]
This dashboard looks great! It's pretty neat!

**Review by Nick-Ohman — APPROVED** [review]
Looks great!

File: src/components/pages/AddChild/RenderAddChild.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ const RenderAddChild = props => {
     <Layout className="add-child">
       <ParentNavSider selected="dashboard" />
 
-      <Layout className="content">
+      <Layout className="child-content">
         <Title className="title" style={{ color: '#0267C1' }} level={1}>
```
Comment (Schrese) [inline src/components/pages/AddChild/RenderAddChild.js:50]: Eventually, we should go through and get rid of all of our in-line styles.

File: src/styles/less/mixins.less [hunk 2]
```diff
@@ -0,0 +1,23 @@
+.welcomeStyle() {
+  padding: 12%;
+  margin-top: 15%;
+  text-align: center;
+  size: 2rem;
+}
+
+.logoStyle() {
```
Comment (Schrese) [inline src/styles/less/mixins.less:8]: Is this for the logo icon?

**Review by Schrese — APPROVED** [review]
Very nice styling. Thanks for adding more mixins as well

**Pull request merged**